lib/v2/screens/settings/edit_name/edit_name_screen.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/v2/screens/settings/edit_name/components/flat_button_long.dart';
+import 'package:seeds/v2/screens/settings/edit_name/components/text_form_field_custom.dart';
+import 'package:seeds/v2/screens/settings/edit_name/viewmodels/bloc.dart';
+
+class EditNameScreen extends StatefulWidget {
+  const EditNameScreen({Key key}) : super(key: key);
+
+  @override
+  _EditNameScreenState createState() => _EditNameScreenState();
+}
+
+class _EditNameScreenState extends State<EditNameScreen> {
+  EditNameBloc _editNameBloc;
+  final _nameController = TextEditingController();
+  final _formKeyPassword = GlobalKey<FormState>();
+
+  @override
+  void initState() {
+    super.initState();
+    _editNameBloc = EditNameBloc(
+        // profileRepository: RepositoryProvider.of<ProfileRepository>(context),
+        );
+    _nameController.text = 'Raul';
+    _nameController.addListener(_onNameChanged);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: AppColors.primary,
+      appBar: AppBar(title: Text('Edit Display Name'), elevation: 0.0),
+      body: Form(
+        key: _formKeyPassword,
+        child: Padding(
+          padding: const EdgeInsets.all(16.0),
+          child: Column(
+            children: [
+              TextFormFieldCustom(
+                controller: _nameController,
+                onFieldSubmitted: (_) => _onSubmitted(),
```

Comment:
commented code needed? 
What are we missing here? 

requierements? Ask in the channel

Reply:
It is logic that I had planned to implement in the following PR

Reply:
kk,

Reply:
It is logical that I do not advance from the flow yet, because I have found the call to update the name. Maybe I should have removed theses comments